modules/exploits/linux/upnp/dlink_dir859_exec_ssdpcgi.rb

```diff
@@ -0,0 +1,95 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
The HTTP methods that you're including here don't look like they're being used anywhere.